spec/controllers/reports/regions_controller_spec.rb

```diff
@@ -550,37 +573,163 @@
       sign_in(cvho.email_authentication)
 
       period = Period.month("July 2018")
-      expect(MonthlyDistrictDataService).to receive(:new).with(region, period, medications_dispensation_enabled: false).and_call_original
-      get :monthly_district_data_report,
-        params: {id: region.slug, report_scope: "district", format: "csv", period: period.value}
+
+      exporter = MonthlyDistrictData::Hypertension.new(
+        region: region,
+        period: period,
+        medications_dispensation_enabled: false
+      )
+      allow(MonthlyDistrictData::Hypertension).to receive(:new).and_return(exporter)
```

Comment:
[Suggestion] A comment about why you need to mock `:new` would be helpful when returning to this test in the future.